.git_archival.txt

```diff
@@ -0,0 +1,4 @@
+node: $Format:%H$
+node-date: $Format:%cI$
+describe-name: $Format:%(describe:tags=true,match=*[0-9]*)$
+ref-names: $Format:%D$
```

Comment:
I guess `ref-names` is not needed? Compare https://setuptools-scm.readthedocs.io/en/latest/usage/ and https://github.com/pypa/setuptools_scm/issues/806 .

Reply:
I'm following the [Hatchling dynamic versioning instructions from here](https://learn.scientific-python.org/development/guides/packaging-simple/)